index.html

```diff
@@ -22,46 +22,40 @@
+
+                <h3 class="persona-desc">Design, Strat, and Stats</h3>
             </div>
             
             <figure class="persona-photo">
-                <img src="img/olana.jpg" alt="">
+                <img src="img/sarima.jpg" alt="">
             </figure>
 
-            <ul class="hats" id="hats-olana">
-                <li class="hat hat-billing">
-                    <i class="haticon haticon-billing"></i> <span class="hat-label">Billing</span>
-                </li>
+            <ul class="hats" id="hats-mark">
                 <li class="hat hat-strategy">
                     <i class="haticon haticon-strategy"></i> <span class="hat-label">Strategy</span>
                 </li>
-                <li class="hat hat-analysis">
-                    <i class="haticon haticon-analysis"></i> <span class="hat-label">Analysis</span>
-                </li>
-                <li class="hat hat-system-health">
-                    <i class="haticon haticon-system-health"></i> <span class="hat-label">System Health</span>
-                </li>
                 <li class="hat hat-implementation">
                     <i class="haticon haticon-implementation"></i> <span class="hat-label">Implementation</span>
                 </li>
+                <li class="hat hat-analysis">
+                    <i class="haticon haticon-analysis"></i> <span class="hat-label">Analysis</span>
+                </li>
             </ul>
 
             <blockquote class="persona-quote">
-                I want to use a web site to build a campaign and watch how it’s doing.
+                Updating a prototype at the click of a button would drastically help turn around time.
             </blockquote>
 
             <div class="persona-details-more">
-                <p>Olana wants to build and run <strong>multiple campaigns, over multiple channels</strong>. He needs to <strong>track the cost</strong> of the campaign, and to <strong>know when things go wrong</strong>, and why. He wants to <strong>see if a campaign is working as expected</strong>, and <strong>see if it's doing well</strong>. He did ICT4D at University a few years ago, so he's <strong>pretty tech-savvy</strong>.</p>
+                <p>Sarima seeks to <strong>understand the needs </strong> of every stakeholder and <strong>design the system</strong> so that it meets stakeholder needs.</p>
 
-                <p>For in-country deploys he wants to <strong>choose the Vumi components that he needs</strong> and just use those. <em>:moonshot:</em></p>
+                <p>Sarima needs to <strong>conduct tests with users</strong> using either paper or clickable prototypes on a mobile device. She needs the ability to <strong>easily update her prototype</strong> in the field and in co-design sessions. She mainly uses her laptop but sees value in making updates via her mobile. She often <strong>exports data and crunches it in Excel</strong> to get the reports she needs.</p>
```

Comment:
> `Sarima needs to <strong>conduct tests with users</strong> using either paper or clickable prototypes on a mobile device`

If I'm remembering correctly from when we chatted to Jacqui, I think paper prototypes are one possible case for when testing on a phone isn't possible, but most of the time (or at least the ideal case) is to test it as an actual USSD service (if the plan is for it to be over USSD) rather than in a web/native app's UI on a mobile.

> `She mainly uses her laptop but sees value in making updates via her mobile`

Wondering if it would be helpful to mention the use case for the mobile (e.g. when co-designing with client).

> `She often <strong>exports data and crunches it in Excel</strong> to get the reports she needs`

Two things:
  - From what I understand from our discussion with Jacqui, I think there important analysis tasks,for e.g. seeing the results of a change they have made the system. Is that included in the reports mentioned here? Wondering if we should mention it somehow?
  - Wondering if we need to mention cases where the service designer would want to see something in a UI/dashboard/graph rather than export and post-process? Maybe things like analysing which screens have high drop-off rates?